### DISCUSSION TITLE
What is the command to view ip routes added by strongswan in MAC

### DISCUSSION THREAD

**Discussion by salmanmalik-emb** [strongswan/strongswan#1756] [Q&A]
In ubuntu, we have a command to view table 220 `ip route list table 220`, what is the equivalent command for MAC to view the routes for table 220. 

I cant see routes or route table added by strongswan by following commands in mac `netstat -rn`.

**Comment by tobiasbrunner** [comment]
If strongSwan installs routes on macOS (or FreeBSD) it uses the main routing table. So the latter command is the correct one to see them. You can increase the log level for _knl_ to see details about the route installation.
